Yoast/javascript
# Add exception checks for irregular nouns to the Russian stemmer

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a check for exception lists of full forms and words with multiple stems to the Russian stemmer.
* [Yoast SEO Premium] Improves word form recognition of Russian nouns that have irregular plurals.

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all current specs pass
* Add more forms of the following Russian nouns to the specs and make sure all forms get the same stem: ухо, черт, человек, ребёнок, друг, сын, колено, небо, чудо, мать, дочь, ку́рица (all forms of a noun can be found by entering the word on this website: http://morpher.ru/Demo.aspx).
* In Russian Premium: enter singular and plural forms of the Russian nouns from the previous step to the keyphrase and to the title/text/meta description and see whether they are recognised.

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-441
